CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.1.11
+Released on January 23, 2023
+
+### Enhancements
+* Uses the elkjs library to improve node and edge positioning on the UI.
+* Shows the name of the options on the UI's menu sidesheet to improve clarity.
+* Removes the Aqueduct logo on the UI's home page to reduce redundancy.
+
+### Bugfixes
+* Fixes bug where operator execution fails when running on Kubernetes.
+* Fixes bug where the workflow details page keeps re-rendering.
+* Fixes bug where the Kubernetes logo doesn't show up on the UI.
+* Fixes bug where the UI keeps hitting the notification route.
```

Comment:
What was causing this repeated query? What was the effect?

Reply:
@vsreekanti For the bug related to the notifications being hit repeatedly, we had a useEffect with no dependency array in layouts/default.tsx, which caused the route to get hit every time something changes.

To remedy this, all we needed to do was include an empty dependency array so that the notifications fetch only happens once.